main/doasedit/template.py

```diff
@@ -1,14 +1,16 @@
 pkgname = "doasedit"
-pkgver = "1.0.7"
+pkgver = "1.0.8"
 pkgrel = 0
+build_style = "makefile"
```

Comment:
The current Makefile does not use `PREFIX`, I've made a merge request to fix this.
https://codeberg.org/TotallyLeGIT/doasedit/pulls/33